src/main/java/org/xbill/DNS/RRset.java

```diff
@@ -35,6 +37,16 @@ public RRset(Record record) {
     addRR(record);
   }
 
+  /** Creates an RRset and sets its contents to the specified record(s) */
+  public RRset(Record... records) {
+    this();
+    if (records != null) {
```

Comment:
The java vararg design is terrible in that it is possible to pass in a null value of type Record[]. While I think this is fine, I like the `Objects.requireNoNull()` construct instead. If someone explicitly feeds this method a `(Record[])null` they deserve a NullPointerException ;)